### ISSUE TITLE
No recount re-skin

### ISSUE THREAD

**Issue by Sourenics** [Azilroka/AddOnSkins#179]
So I recently installed the addon. On the Skin tab all addons are marked by default except for DBM and Recount. I check the box and everything seems fine. Addon asks me to reload the interface. When I do it, check again the Skin tab and Recount it's unmakerd. Thing is, no matter how many times I check the box and reload, it keeps unmarking itself and recount does not apply the skin.

**Comment by Azilroka** [comment]
That means no matter how many times you do this you don't see the message in chat saying there is an error in the skin. It automatically disables skins that has errors.

**Comment by Sourenics** [comment]
I noted that if i delete recount and addonskins config inside the WTF/Account/<account>/SavedVariables/ and then log into the game, the recount skin changes, but as soon as you reload the UI (using /reload, entering dungeon, etc.) it goes back to normal and can't change it anymore.

**Comment by Sourenics** [comment]
If I enable the skin and reload the UI it shows this:
`Message: Interface/AddOns/AddOnSkins/Core/Skins.lua:1654: attempt to index local 'frame' (a nil value)
Time: Tue Aug 29 22:11:24 2023
Count: 1
Stack: Interface/AddOns/AddOnSkins/Core/Skins.lua:1654: attempt to index local 'frame' (a nil value)
[string "=(tail call)"]: ?
[string "@Interface/AddOns/AddOnSkins/Core/Skins.lua"]:1654: in function `Desaturate'
[string "@Interface/AddOns/AddOnSkins/Skins/AddOns/Recount.lua"]:20: in function `SkinFrame'
[string "@Interface/AddOns/AddOnSkins/Skins/AddOns/Recount.lua"]:25: in function <Interface/AddOns/AddOnSkins/Skins/AddOns/Recount.lua:5>
[string "@Interface/AddOns/AddOnSkins/Core/Core.lua"]:182: in function <Interface/AddOns/AddOnSkins/Core/Core.lua:182>
[string "=[C]"]: in function `xpcall'
[string "@Interface/AddOns/AddOnSkins/Core/Core.lua"]:182: in function `CallSkin'
[string "@Interface/AddOns/AddOnSkins/Core/Core.lua"]:123: in function `SkinEvent'
[string "@Interface/AddOns/AddOnSkins/Core/Core.lua"]:263: in function `?'
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:109: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...UI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals: (*temporary) = <function> defined =[C]:-1
`

**Comment by Azilroka** [comment]
Yes by enabling Skin Debug that will allow the skin to keep running regardless if there is an error. Which means line 25 is the culprit of the Skin disabling itself with Skin Debug turned off.
